pkg/templates/nstemplatetiers/nstemplatetier_generator_test.go

```diff
@@ -206,3 +215,12 @@ func verifyTierTemplate(t *testing.T, cl *commontest.FakeClient, namespace, tier
 	require.Fail(t, fmt.Sprintf("the TierTemplate for NSTemplateTier '%s' and of the type '%s' wasn't found", tierName, typeName))
 	return ""
 }
+
+func newTierTemplate(name, ns string) *toolchainv1alpha1.TierTemplate {
```

Comment:
minor - this function looks like a duplicate of what we already have in `nstemplatetier_controller_test.go` and `tiertemplate_guard_controller_test.go`. I'm wondering if we could extract it to common function ( if there is one already ).